src/xma/src/xmaplugin/xmaplugin.cpp

```diff
@@ -31,32 +31,60 @@ using namespace std;
 
 #define XMAPLUGIN_MOD "xmapluginlib"
 
-//Sarab: TODO .. Assign buffr object fields.. bank etc..
-//Add new API for allocating device only buffer object.. which will not have host mappped buffer.. This could be used for zero copy plugins..
-//NULL data pointer in buffer obj implies it is device only buffer..
-//Remove read/write before SyncBO.. As plugin should manage that using host mapped data pointer..
+
+int32_t create_bo(xclDeviceHandle dev_handle, XmaBufferObj& b_obj, uint32_t size, uint32_t ddr_bank, 
+    bool device_only_buffer, xclBufferHandle& b_obj_handle) {
+    /*
+    #define XRT_BO_FLAGS_MEMIDX_MASK        (0xFFFFFFUL)
```

Comment:
These define statements looks like "helper comments" for future folks.  If the later, you might wish to add a comment for the future self :^) regarding the purpose behind these values.  If not, then I would recommend removing them.

Reply:
:) done